test/explorer/data_frame_test.exs

```diff
@@ -4804,10 +4807,12 @@ defmodule Explorer.DataFrameTest do
       end
     end
 
-    @tag :skip
-    property "can dump any DataFrame to NDJSON" do
+    property "can dump any non-binary, non-time DataFrame to NDJSON" do
+      not_yet_implemented = [:binary, :time]
+
       check all(
-              dtypes <- Explorer.Generator.dtypes(),
+              # TODO: Remove `exclude: :category` after #1011 is resolved.
+              dtypes <- Explorer.Generator.dtypes(exclude: [:category | not_yet_implemented]),
```

Comment:
If we don't exclude e.g. `:time`, we get:

```
thread '<unnamed>' panicked at path/to/.cargo/registry/src/index.crates.io-6f17d22bba15001f/polars-json-0.43.1/src/json/write/serialize.rs:517:18:
not yet implemented: Writing Time64(Nanosecond) to JSON
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

It may be better to check for the not-yet-implemented dtypes in Elixir and raise an `ArgumentError` if we find them. We could do this either in `Explorer.DataFrame` or `Explorer.PolarsBackend.DataFrame`.

Thoughts?

Reply:
It needs to be done in the backend I think because the decision is backend specific. I can see some of them happy in converting Time64 to an integer or a string to send as JSON.

Reply:
Yeah I think so.

I'd love to document it too, otherwise it'll be a bit of a surprise. I'll see if I can find a good place for it.